pex/bin/pex.py

```diff
@@ -290,8 +293,11 @@ def configure_clp_pex_environment(parser):
   group.add_option(
       '--platform',
       dest='platform',
-      default=Platform.current(),
-      help='The platform for which to build the PEX.  Default: %default')
+      default=[],
```

Comment:
ah, how would you feel about just replacing the `%default` here with a literal description?

the problem with defaulting to `[Platform.current()]` here is that the `action=append` mode is additive only, so without implementing a custom callback action for this option you'd end up resolving/building for the local platform every time whether or not you actually intended to.